### PR TITLE
Extract WebAuthnPublicKey and WebAuthnSignature to separate files

### PR DESCRIPTION
**Pull request by akonior**

### CODE REVIEW

File: src/app/account/WebAuthnSignature.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const CLIENT_DATA_JSON_MAX_LEN = 255;
+
+export class WebAuthnSignature {
```
Comment (marekkirejczyk) [inline src/app/account/WebAuthnSignature.ts:3]: Move to `model/webauthn`?

File: src/app/account/webauth_sign_challange.ts [hunk 2]
```diff
@@ -2,7 +2,11 @@ import { convertBigIntToUint8Array, convertUint8ArrayToBigInt } from './utils.js
 
 const SUPPORTED_CLIENT_DATA_JSON_LENGTHS = [114, 134, 243];
 
-export async function webAuthnSignChallenge(challenge: Uint8Array) {
+export async function webAuthnSignChallenge(challenge: Uint8Array): Promise<{
```
Comment (marekkirejczyk) [inline src/app/account/webauth_sign_challange.ts:5]: Move to `infra/webauthn`

File: src/app/account/webauthn_account_contract.ts [hunk 3]
```diff
@@ -11,8 +11,8 @@ import {
 } from '@aztec/aztec.js';
 
 import { AuthWitness } from '@aztec/types';
-
-const CLIENT_DATA_JSON_MAX_LEN = 255;
+import { WebAuthnPublicKey } from './WebAuthnPublicKey.js';
+import { WebAuthnSignature } from './WebAuthnSignature.js';
```
Comment (marekkirejczyk) [inline src/app/account/webauthn_account_contract.ts:15]: Moce to `infra/aztec`

File: src/app/account/webauthn_signer.ts [hunk 4]
```diff
@@ -1,6 +1,8 @@
-import { WebAuthnInterface, WebAuthnPublicKey, WebAuthnSignature } from './webauthn_account_contract.js';
+import { WebAuthnInterface } from './webauthn_account_contract.js';
 import { webauthnCreatePublicKey } from './webauthn_create_public_key.js';
 import { webAuthnSignChallenge } from './webauth_sign_challange.js';
+import { WebAuthnPublicKey } from './WebAuthnPublicKey.js';
+import { WebAuthnSignature } from './WebAuthnSignature.js';
```
Comment (marekkirejczyk) [inline src/app/account/webauthn_signer.ts:5]: Move to `infra/webauthn`

**Review by marekkirejczyk — DISMISSED** [review]

File: src/app/account/WebAuthnPublicKey.ts [hunk 5]
```diff
@@ -0,0 +1,6 @@
+export class WebAuthnPublicKey {
+  constructor(
+    readonly x: Uint8Array,
+    readonly y: Uint8Array,
+  ) {}
+}
```
[the remark this answers is not in the document]
Comment (akonior) [reply src/app/account/WebAuthnPublicKey.ts]: Done

File: src/app/account/WebAuthnSignature.ts (hunk 1 above)
Comment (akonior) [reply src/app/account/WebAuthnSignature.ts]: done

File: src/app/account/webauth_sign_challange.ts (hunk 2 above)
Comment (akonior) [reply src/app/account/webauth_sign_challange.ts:5]: done

File: src/app/account/webauthn_account_contract.ts (hunk 3 above)
Comment (akonior) [reply src/app/account/webauthn_account_contract.ts]: done

File: src/app/account/webauthn_signer.ts (hunk 4 above)
Comment (akonior) [reply src/app/account/webauthn_signer.ts]: done

**Pull request merged**